### PR TITLE
made events more dinamically

### PR DESCRIPTION
**Pull request by mbiarnes**

### CODE REVIEW

File: _config/events.yml [hunk 1]
```diff
@@ -17,3 +17,24 @@
...
+  title: Hackergarten
+  url: http://www.devoxx.be
+
+- eventId: 20161122-jbuglondon
+  eventOrganization: JBoss User Group
+  location: London, UK
+  date: 2016-11-22
+  title: Red Hat BRMS and BPMS Roadmap Presentation
+  url: http://www.meetup.com/JBoss-User-Group/events/234803765/
+  presenters: Geoffrey De Smet, Mark Proctor, Kris Verlaenen, Mauricio Salatino, Max Barkley
```
Comment (ge0ffrey) [inline _config/events.yml]: Change order to put Kris, Mauricio's and Mark name first as this is jbpm.org

**Comment by mbiarnes** [comment]
done

File: community/eventsArchive.html.haml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: Events archive
+description: A list of all past and future OptaPlanner events.
```
Comment (psiroky) [inline community/eventsArchive.html.haml]: Just noticed this now - this should be jBPM.

Comment (mbiarnes) [inline community/eventsArchive.html.haml]: will fix this ASAP

**Comment by krisv** [comment]
Merged

**Pull request closed without merging**